run-golangci-lint.sh

```diff
@@ -1,2 +1,9 @@
 #!/usr/bin/env sh
+
+if command -v golangci-lint &> /dev/null ; then
```

Comment:
Oh oops, did I merge this too quickly? Should these be `! command -v ` ? I'll fix this in a follow up